src/views/UserDetail/UserDetail.js

```diff
@@ -431,6 +523,8 @@ class UserDetail extends React.Component {
     } = this.state;
 
     const user = this.getUser();
+    const username = getFullName(user);
+    const userId = this.props.match.params.id;
```

Comment:
`match` is already destructured. You can use it directly here: `const userId = match.params.id;`